.snippets/code/ethereum-api/web3py/balances.py

```diff
@@ -0,0 +1,20 @@
+# 1. Import web3.py
+from web3 import Web3
+
+# 2. Create web3.py provider
+provider_rpc = {
+    # Insert your RPC URL here
+    'evm_containerchain': 'https://fraa-dancebox-3001-rpc.a.dancebox.tanssi.network',
+}
+web3 = Web3(Web3.HTTPProvider(provider_rpc['evm_containerchain']))
+
+# 2. Create address variables
+address_from = 'ADDRESS_FROM_HERE'
```

Comment:
Made this comment on the Ethers PR, but variable placeholders should start with `INSERT` following by whatever you're using to describe the variable and they should not end with `HERE`

so this should be `INSERT_FROM_ADDRESS` (made this change on the moonbeam-docs repo already)